src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -7437,6 +7444,16 @@
 			path = HealthCertifiedPerson;
 			sourceTree = "<group>";
 		};
+		8FE68F9128A1469E003E86E1 /* ELS */ = {
+			isa = PBXGroup;
+			children = (
+				8FE68F9228A146CC003E86E1 /* SubmitELSResource.swift */,
```

Comment:
Can you please rename this and the corresponding files to ELSSubmitResource to align with PPASubmitResource